modules/kms/main.tf

```
@@ -1,15 +1,78 @@
+locals {
+  account_id        = data.aws_caller_identity.current.account_id
+  account_principal = "arn:${data.aws_partition.current.partition}:iam::${local.account_id}:root"
+
+  principals = sort(distinct(concat(
```

I'd suggest keeping the original name so it's clearer in the kms policy

```suggestion
  allowed_principal_arns = sort(distinct(concat(
```

or making it even clearer

```suggestion
  kms_usage_principal_arns = sort(distinct(concat(
```

---

Ignored